src/utils.ts

```diff
@@ -8,6 +8,10 @@ export function isObject(value: unknown): value is object {
   return value !== null && typeof value === 'object';
 }
 
+export function isTypeUndefined(value: unknown): value is object {
```

Comment:
```suggestion
export function isTypeUndefined(value: unknown): value is undefined {
```
^ Copy-paste error I think?

Reply:
@jlefebure Commit this if I'm right, or just let me know if I'm missing something. Then I'll merge!! :D

Reply:
Woops, yes it is